scalardb-sample/docker-compose.yml

```diff
@@ -1,25 +1,41 @@
 version: "3.5"
```

Comment:
```suggestion
```
I LGTMed but I forgot we can remove this otherwise since it is obsolete.
```
WARN[0000] /Users/vguilpain/dev/scalardb-samples/scalardb-sample/docker-compose.yml: `version` is obsolete
```

Reply:
Thank you. Since it should also be applied to the other docker compose files in other samples, I will make the change in a separate PR. Thanks.